### ISSUE TITLE
Resume from sleep renders file system `read only`

### ISSUE THREAD

**Issue by nuke-web3** [pop-os/pop#2671]
**Distribution**
```
 NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os
 ```
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
(all)

**Issue/Bug Description:**

Most (but seemingly not all times) a 17" Gazelle with NVIDIA GeForce RTX 3060 I am running will resume from sleep but run into issue with interation with any app.

**Steps to reproduce (if you know):**

1. Close lid, assume the device goes to sleep
1. Resume, navigate between windows (works)
1. Try to make any change on disk (doesn't work, user or system origin)
1. Try to shutdown/restart the system normally, no effect, no errors.
1. Hard power off (hold button) to restart

**Other Notes:**

- Been acting this way from the beginning, but not a PITA enough till I need to move the machine more regularly working remotely.
- I have 2 SSDs in the computer, I doubt that confusion about what one would be an issue, the 2nd is never mounted. 
- I am using integrated graphics only, but I believe hybrid or dedicated GPU only has same behavior. 

Related maybe? #2465 #2641

---

Will try and take better notes next time it happens manually, as I cannot save to the machine itself when it happens :sweat_smile: .

**Comment by leviport** [comment]
Can you share the model numbers for the SSDs that are currently installed? This usually has to do with certain drives doing weird things when they try to enter power saving states, either during suspend cycles or during idle/inactivity.

**Comment by LBWright** [comment]
I am also having this issue on the brand new Lemur Pro, shipped with factory basics and defaults.

**Comment by leviport** [comment]
@LBWright Can you share the model of drive that this is happening with? You should be able to see that info with the Disks utility, or with `lspci -v` if you installed a different distro than it shipped with.

**Comment by leviport** [comment]
> `01:00.0 Non-Volatile memory controller: Sandisk Corp WD Blue SN570 NVMe SSD (prog-if 02 [NVM Express])`

Yep, that's the kind we had the most issue with. [ASPM](https://en.wikipedia.org/wiki/Active_State_Power_Management) turns those WD Blue drives off for power savings, and something about them fails to return from the low-power state afterwards. If this is a drive that came with the machine from System76, please open a [support ticket](https://support.system76.com/) and we will help get this sorted out.

**Comment by jacobgkau** [comment]
Hello, do either of you still need assistance with this, or was the support team able to get it resolved for you?

**Comment by jacobgkau** [comment]
Closing due to no further contact. Feel free to comment again, open a new issue, or [contact support](https://support.system76.com) if you still need assistance.
